tests/unit/utils/I18N.spec.ts

```diff
@@ -59,8 +59,11 @@ describe('I18N', () => {
           const date = new Date('2023-04-13T00:35:46.789Z')
           const result = i18n.formatDateAsTime(date)
 
-          const timeRegex = /0?0[:.]35|12[:.]35/
-          expect(result).toMatch(timeRegex)
+          if( language != "fa"){
```

Comment:
I don't like this hidden bug. Let's rather just "disable" the fa translation so someone can come in and fix it.

Reply:
Not sure if this is really a bug, because something is translated

Reply:
Revert it and changed the encoding of the file to UTF-8